src/passes/SyntaxHighlighter.jl

```diff
@@ -123,7 +123,7 @@ function (highlighter::SyntaxHighlighterSettings)(crayons::Vector{Crayon}, token
                 crayons[i] = cscheme.keyword
             end
         # "foo"
-        elseif kind(t) == Tokens.STRING || kind(t) == Tokens.TRIPLE_STRING || kind(t) == Tokens.CHAR
+        elseif kind(t) == Tokens.STRING || kind(t) == Tokens.TRIPLE_STRING || kind(t) == Tokens.CHAR || kind(t) == Tokens.CMD
```

Comment:
`TRIPLE_CMD` as well?

Reply:
forgot about that. Just added. Also added backtick matching similar to single and double quotes.